modules/core/src/main/java/org/apache/ignite/spi/discovery/tcp/ServerImpl.java

```diff
@@ -4737,7 +4741,7 @@ else if (!locNodeId.equals(node.id()) && ring.node(node.id()) != null) {
                                 spi.marshaller(), U.resolveClassLoader(spi.ignite().configuration()), node
                             );
 
-                            if (!permissionsEqual(coordSubj.subject().permissions(), subj.subject().permissions())) {
+                            if (!permissionsEqual(getPermissions(coordSubj), getPermissions(subj))) {
```

Comment:
Why we only fix one usage of this method? Are `null`'s possible in another branch?

Reply:
No, other method usages contain null check.

Reply:
Where is that check? I see only usage of the same method `nodeSecurityContext` to get `coordSubj` and no null check after it.